app/ui/shared/widgets/autocompleted-search.jsx

```diff
@@ -47,20 +47,26 @@ class AutocompletedSearch extends Component {
 
   componentWillMount() {
     if (typeof window === 'object') {
-      window.addEventListener('click', this.handleWindowClick);
+      window.addEventListener('mousedown', this.handleWindowMousedown);
     }
   }
 
   componentWillUnmount() {
     if (typeof window === 'object') {
-      window.removeEventListener('click', this.handleWindowClick);
+      window.removeEventListener('mousedown', this.handleWindowMousedown);
     }
   }
 
-  handleWindowClick = () => {
-    // We can't call `setState` on components which haven't rendered yet.
-    if (this.state.showSelectionList && this.inputbar) {
+  // We need to use mousedown since preventing the event on click won't stop
+  // it from passing through to the webview.
```

Comment:
Not sure I understand this comment. Preventing click events won't stop the click events from propagating? Am I reading this right? Why "prevent" and not "stop propagation"?